api/proto/info/info.proto

```diff
@@ -333,21 +433,65 @@ message ZInfoStorage {
   bool storageLocation = 4; // Storage location for app disks, images etc.
 }
 
+// ZInfoAppNetworkInterface
```

Comment:
I think it is easier to reuse ZInfoNetwork for the applications, and mark the To-be-deprecated/unused fields as such.
Ideally they are not even used by the controller (since they are never filled in by EVE) so we might be able to remove them. But that is a lot less pain that introducing a new message and at some later point remembering to delete the old message definition and code.